php-frontend/src/main/java/org/sonar/plugins/php/api/tree/Tree.java

```diff
@@ -196,9 +196,14 @@
     PARENTHESISED_EXPRESSION(ParenthesisedExpressionTree.class),
 
     /**
-     * {@link ParenthesisedExpressionTree}
+     * {@link YieldExpressionTree}
+     */
+    YIELD_EXPRESSION(YieldExpressionTree.class),
```

Comment:
This needs to be documented in the ticket, and ideally in the javadoc: "since 3.0"

Reply:
I documented the change without mentioning ParenthesisedExpressionTree, because it was previously wrong.